### ISSUE TITLE
Option to load message history

### ISSUE THREAD

**Issue by MurKit** [yakyak/yakyak#715]
Now I can see only one last message in the conversation. 
Scrolling up does not load previous messages, and there is no obvious button for doing this.
Am I missing something? Or this should be a feature request?

**Comment by averissimo** [comment]
scrolling up should do this, does this happen to all your conversations?

**Comment by MurKit** [comment]
Yeah, it feels like scrolling up should do this, but when the latest showed part of conversation is too small to scroll somewhere, older messages just don't load.

**Comment by sriharshachilakapati** [comment]
This one is still active. Any updates on this?
